pom.xml

```diff
@@ -9,7 +9,7 @@
   <parent>
     <groupId>app.coronawarn</groupId>
     <artifactId>cwa-parent</artifactId>
-    <version>1.3</version>
+    <version>1.4</version>
```

Comment:
suggest to change to \<version>[1,2.0]\</version> to include always latest version of parent (up to but not including 2.0)

Reply:
In favor of reproducible builds this is a bad idea